test/deploy/operator.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         - name: managed-upgrade-operator
           # Replace this with the built image name
           # This will get replaced on deploy by /hack/generate-operator-bundle.py
-          image: quay.io/app-sre/managed-upgrade-operator:latest
+          image: quay.io/app-sre/managed-upgrade-operator:v0.1.809-46d986a
```

Comment:
Similarly I feel like this benefits from being an invalid tag because it forces the SRE testing the operator to think about "what version do I _want_ to test?", rather than not thinking about that at all and having some old version be used.

Reply:
I've had a go at pushing this change to the branch.